src/ruby/yast/builtins.rb

```diff
@@ -366,13 +366,11 @@ def self.swap(list, offset1, offset2)
         return Yast.deep_copy(list) if offset1 < 0 || offset2 >= list.size || (offset1 > offset2)
 
         res = []
-        if offset1 > 0
-          res.concat list[0..offset1 - 1]
-        end
-        res.concat list[offset1..offset2].reverse!
-        if offset2 < list.size - 1
-          res.concat list[offset2 + 1..-1]
-        end
+
+        res.concat(list[0..offset1 - 1]) if offset1 > 0
+        res.concat(list[offset1..offset2].reverse!)
+        res.concat(list[offset2 + 1..-1]) if offset2 < list.size - 1
```

Comment:
Builtins::List.swap is used only twice in a single file; we could remove it:
https://github.com/yast/yast-storage/blob/3e2a3edaa59c1e6d7c8805d2e82e017ea496eebd/src/modules/DualMultiSelectionBox.rb#L604